src/main/scala/tilelink/ToAXI4.scala

```diff
@@ -149,8 +149,8 @@ class TLToAXI4(val combinational: Boolean = true, val adapterName: Option[String
       val depth = if (combinational) 1 else 2
       val out_arw = Wire(Decoupled(new AXI4BundleARW(out.params)))
       val out_w = Wire(chiselTypeOf(out.w))
-      out.w :<> Queue.irrevocable(out_w, entries=depth, flow=combinational)
-      val queue_arw = Queue.irrevocable(out_arw, entries=depth, flow=combinational)
+      out.w :<> Queue.irrevocable(out_w, entries=depth, combinational)
```

Comment:
Is the deletion of `flow=` intended? @sequencer It seems we must pass the argument explicitly by `flow=` unless this argument would be used as `pipe` instead of `flow`.

In https://github.com/chipsalliance/chisel/blame/main/src/main/scala/chisel3/util/Decoupled.scala#L394-L401
```scala
  @chiselName
  def irrevocable[T <: Data](
    enq:            ReadyValidIO[T],
    entries:        Int = 2,
    pipe:           Boolean = false,
    flow:           Boolean = false,
    useSyncReadMem: Boolean = false,
    flush:          Option[Bool] = None
  ): IrrevocableIO[T] = {
...
}
```

We have faced performance regression after bumping rocket to 0586532e47b679d472bed5dd2c70c246ba1da0c6 and are working on finding the root cause. Currently I'm seeing the Verilog differences because of this Queue. Still waiting for the XiangShan CI to see whether fixing this could address the performance issue. But up to now this line is quite suspective.

Reply:
After reverting this line, the Verilog seems to be aligned with the same number of Queue* modules generated. The performance is to be tested at https://github.com/OpenXiangShan/XiangShan/pull/2353. Will see the results this afternoon

Reply:
This is certainly a bug. please PR ;p

Reply:
Submitted